sdk/core/core-rest-pipeline/review/core-rest-pipeline.api.md

```diff
@@ -307,6 +317,37 @@ export interface RestErrorOptions {
     statusCode?: number;
 }
 
+// @public
+export interface RetryInformation {
+    response?: PipelineResponse;
+    responseError?: RestError;
+    retryCount: number;
+}
+
+// @public
+export interface RetryModifiers {
+    redirectTo?: string;
+    retryAfterInMs?: number;
+    skipStrategy?: boolean;
+    throwError?: RestError;
```

Comment:
I couldn't decide: `throwError` feels more like a boolean so `errorToThrow` may be better but I don't know whether it is worth.

Reply:
Oohh I like that! I’ll apply it!

Reply:
Updated here: https://github.com/Azure/azure-sdk-for-js/pull/19078/commits/93b3aaff10025bb684e6edbc5d38835df3adf998